bedshift/bedshift.py

```diff
@@ -446,8 +555,12 @@ def main():
 
     _LOGGER.info("Welcome to bedshift version {}".format(__version__))
     _LOGGER.info("Shifting file: '{}'".format(args.bedfile))
+    
+    if args.yaml:
```

Comment:
As discussed on Messenger, we can integrate this better with the existing code on line 608